cookienotice/cookienotice.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Name: Cookie Notice
+ * Description: Configure, show and handle a simple cookie notice
+ * Version: 1.0
+ * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
+ * 
+ */
+use Friendica\Core\Addon;
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+
+function cookienotice_install()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
+	Addon::registerHook('page_content_top', $file, 'cookienotice_page_content_top');
```

Comment:
In the upcoming release this method has been deprecated in favor of `Hook::register`. Since we are so close from the release date I will ask you to update all calls from `Addon::registerHook` to `Hook::register` and `Addon::unregisterHook` to `Hook::unregister`.

Reply:
Thank you. Saw that already in the develop branch in other addons. I will change it.